.github/workflows/update-flake.yml

```diff
@@ -41,22 +41,32 @@ jobs:
             email "$id+$name@users.noreply.github.com" \
             >>"$GITHUB_OUTPUT"
 
-      - uses: DeterminateSystems/update-flake-lock@v26
-        with:
-          token: ${{ steps.generate-token.outputs.token }}
-          git-committer-name: ${{ steps.user-info.outputs.name }}
-          git-committer-email: ${{ steps.user-info.outputs.email }}
-          git-author-name: ${{ steps.user-info.outputs.name }}
-          git-author-email: ${{ steps.user-info.outputs.email }}
+      - name: setup git
+        env:
+          name: ${{ steps.user-info.outputs.name }}
+          email: ${{ steps.user-info.outputs.email }}
+        run: |
+          git config --global user.name "$name"
+          git config --global user.email "$email"
+
+      - name: update lock files
+        run: |
+          nix flake update \
+            --commit-lock-file \
+            --option commit-lock-file-summary "flake: update root inputs"
+
+          nix flake update \
+            --commit-lock-file \
+            --flake ./flake/dev \
+            --option commit-lock-file-summary "flake: update dev inputs"
+
+      - name: create pull request
+        env:
+          GH_TOKEN: ${{ steps.generate-token.outputs.token }}
+          title: "${{ startsWith(matrix.branch, 'release') && format('[{0}] ', matrix.branch) || '' }}flake: update public and dev inputs"  # yamllint disable-line rule:line-length
           branch: update_flake_lock_action_${{ matrix.branch }}
-          commit-msg: "flake: update all inputs"
-          pr-title: "${{ startsWith(matrix.branch, 'release') && format('[{0}] ', matrix.branch) || '' }}stylix: update all flake inputs"   # yamllint disable-line rule:line-length
-          pr-labels: "topic: dependencies"
-          pr-body: >
-            Automated changes by the
-            [update-flake-lock](https://github.com/DeterminateSystems/update-flake-lock)
-            GitHub Action.
-
-            ```
-            {{ env.GIT_COMMIT_MESSAGE }}
-            ```
+        run: |
+          gh pr create \
+            --base "$branch" \
```

Comment:
Base branch is the branch you want to merge into.

However, currently you're passing `$branch` as the branch you want to use for the PR.

I believe what we want here is:

```sh
git switch --create "$pr_branch"
git push origin "$pr_branch" \
  --force \
  --set-upstream
gh pr create \
  --base "$base_branch"
  #...
```